mindmeld/models/tagger_models.py

```diff
@@ -131,9 +131,14 @@ def fit(self, examples, labels, params=None):
         # Shuffle to prevent order effects
         indices = list(range(len(labels)))
         random.shuffle(indices)
-        examples = [examples[i] for i in indices]
-        labels = [labels[i] for i in indices]
-
+        try:
+            examples.reorder(indices)
```

Comment:
Since this assumes a non-list datatype, can we update this functions description for the type for `examples `